stringlab.js

```diff
@@ -0,0 +1,75 @@
+// console.log the first character of a string.
+let char = 'Hello'
+console.log('The last character of my string is : ' + char[0])
+
+// console.log the length of a string.
+console.log('The length of my string is ' + char.length)
+
+// console.log the last character of any string.
+console.log('The last character of my string is : ' + char[char.length - 1])
+
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+
+let name = 'jimmy'
```

Comment:
What would happen if I change name variable to "kadijah"? Would it still give us last letter capitalized? Your code must be able to accommodate for different inputs and still returns consistent outputs. Always test your code with different inputs next time and see if you can get expected result.